homeassistant/components/tuya/sensor.py

```diff
@@ -889,6 +889,16 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             state_class=SensorStateClass.MEASUREMENT,
         ),
     ),
+    # Smart Gardening system
+    # https://developer.tuya.com/en/docs/iot/categorysz?id=Kaiuz4e6h7up0
+    "sz": (
```

Comment:
Nice! Looking at the documentation/specification, I think we should add the current humidity as well:

![CleanShot 2024-05-29 at 08 40 24@2x](https://github.com/home-assistant/core/assets/195327/cf506a12-04af-4bb2-84ce-1e6ce41b6b46)

Reply:
My device didn't send this value, so I can't actually test it locally if it's work but I can add if you think that's ok.